src/components/FormGroup.jsx

```diff
@@ -1,17 +1,35 @@
 import React from 'react';
 
 const FormGroup = (props) => {
-  const { label, ...rest } = props
+  console.log(props);
+  const { label, ...rest } = props;
   const { styles } = FormGroup;
 
+  function inputField() {
+    if (props.placeholder === 'what are you looking for?') {
```

Comment:
I tried adding a prop **Dashboard={true}** to the FormGroup component nested inside of Dashboard.jsx on line 103, but it was throwing an error in the console. I was going to render it conditionally like so:

if (props.dashboard) {
  return (
    // black text input fields 
  )
} else {
   // white text input fields 
}

Instead, could we add one new prop to FormGroup called blackText and render the input field with specific color conditionally? This in my opinion would be a more full proof implementation. 

Example: 

function inputField() {
   if (props.blackText) {
      // make input text black
   } else {
     // make input text white
   }
}